javascript/javascript2/week1/HYF Bay/main.js

```diff
@@ -0,0 +1,24 @@
+console.log("Script loaded");
+const products = getAvailableProducts();
+console.log(products);
+
+const productUl = document.querySelector('ul');
+
+    function renderProducts (products) {
+        products.forEach(function (product) {
+        const productLi = document.createElement('li');
+        productUl.appendChild(productLi);
+            productLi.innerText = product.name;
+        const itemUl = document.createElement('ul');
+        itemUl.setAttribute('class', 'item-ul')
+            productLi.appendChild(itemUl);
+            const keys = Object.keys(product);
```

Comment:
instead you can make it a bit cleaner by just choosing the keys explicitly:
```
const keys = ['price', 'rating']
```
but that's just my taste

Reply:
Thank you for all your valuable comments, I'll edit them accordingly. Thanks again.